src/components/TopBar.js

```diff
@@ -339,6 +338,7 @@ export default _.flow(
               icon('fcIconWhite', { className: 'is-solid', size: 20 })
             ]), 'Use Classic FireCloud'
           ]),
+          div({ style: { borderTop: `1px solid ${colors.dark(0.55)}` } }, []),
```

Comment:
Though adding a `div` at the end of the menu is a *less* elegant solution to adding a bottom border, it seems like the simplest and easiest to maintain in the future. Due to the different states a user can be in, it can be complicated to determine where to show the border bottom, e.g. on FireCloud.terra.bio, the button to "Use Classic FC" is last, but on app.terra.bio the "Terra Support" dropdown is last.

Reply:
It's pretty standard to bake in the border on either the top or bottom and then special case the other end. The choice of top or bottom depends on the specific design. This is totally fine.

TOL: It looks like the profile box, being the topmost one, has an exception where it omits the top border. Therefore, there are now exceptions on both ends. I wonder if the standard should be to have a bottom border instead of a top border, thereby eliminating both exceptions.

Reply:
That's a good point, I think there's too much going on to make it feasible though.
The issue that I ran into is that since there are sub-items below the section, the bottom border has to move down if it is a `bottomBorder` on the last section.
<img width="290" alt="Screen Shot 2019-05-29 at 10 11 06 AM" src="https://user-images.githubusercontent.com/2395211/58564039-3acf9300-81fa-11e9-89cc-ee1e342fd4e6.png">

compared to:
<img width="291" alt="Screen Shot 2019-05-29 at 10 11 31 AM" src="https://user-images.githubusercontent.com/2395211/58564095-55a20780-81fa-11e9-9945-59a5293c23ad.png">

Reply:
💯 Good call. I didn't think of that.